plugin-server/src/utils/utils.ts

```diff
@@ -61,15 +65,18 @@ export function base64StringToUint32ArrayLE(base64: string): Uint32Array {
 }
 
 export function uint32ArrayLEToBase64String(uint32Array: Uint32Array): string {
```

Comment:
You seem to have removed a lot of places where we were using these conversion functions. Are we still using them somewhere?

Reply:
Removed! (they were used in tests but I just changed the tests)